### ISSUE TITLE
PrettyQr can only set elementColor but not "background color"

### ISSUE THREAD

**Issue by hyUrbanManager** [promops/flutter_pretty_qr#23]
I need to set qr code background color traspant. but api only support set elementColor. in dart file PrettyQrCodePainter _paintRound method
```
 var _paintBackground = Paint()
      ..style = PaintingStyle.fill
      ..color = Colors.white
      ..isAntiAlias = true;
```
Background u set Colors.white, i canot change.
can u add this api? thanks

**Comment by promops** [comment]
@hyUrbanManager  HI! What version are we talking about? 
in 3.0.0 - https://pub.dev/packages/pretty_qr_code/versions/3.0.0 :

`   final paint = Paint()
      ..color = color
      ..isAntiAlias = true
      ..style = PaintingStyle.fill;
 `

**Comment by hyUrbanManager** [comment]
> @hyUrbanManager HI! What version are we talking about? in 3.0.0 background is transparent by default - https://pub.dev/packages/pretty_qr_code/versions/3.0.0 :
> 
> ` final paint = Paint() ..color = color ..isAntiAlias = true ..style = PaintingStyle.fill;`
> 
> I think this problem can be fixed after update to 3.0.0. This version has a **PrettyQr** widget for backward compatibility, or you can use new api.

Thanks for your reply.  I use 2.0.3, because I refer to this URL https://pub-web.flutter-io.cn/packages/pretty_qr_code/install which say latest version is 2.0.3.  now I upgrade to 3.0.0 refer to your url.
I solve my problem, qr code background is transparent, it works!
